tests/backend_test.py

```diff
@@ -1002,30 +1007,14 @@ def test_compilation_correctness(brisbane_backend: IBMQBackend) -> None:
     c.Rz(0.125, 2).X(2).Rz(0.25, 2)
     c.SX(3).Rz(0.125, 3).SX(3)
     c.CX(0, 3).CX(0, 4)
-    u_backend = AerUnitaryBackend()
     c.remove_blank_wires()
     FlattenRelabelRegistersPass().apply(c)
-    u = u_backend.run_circuit(c).get_unitary()
-    ibm_backend = brisbane_backend
+    c_pred = ConnectivityPredicate(brisbane_backend.backend_info.architecture)
```

Comment:
This line still has an issues in the mypy check, could you please fix that or add an ignore?

Reply:
We should maybe add an issues to pytket that the predicate should accept fully connected, too?

Reply:
```suggestion
    c_pred = ConnectivityPredicate(brisbane_backend.backend_info.architecture)  # type: ignore

Reply:
I've cast the object to `Architecture` which worked locally